docfx/doxygen2yaml.cc

```diff
@@ -42,6 +42,24 @@ std::string Summary(pugi::xml_node const& node) {
 
 }  // namespace
 
+bool IncludeInPublicDocuments(pugi::xml_node const& node) {
+  // We do not generate documents for files and directories.
+  if (kind(node) == "file" || kind(node) == "dir") return false;
+  // Doxygen groups private attributes / functions in <sectiondef> elements of
+  // these kinds:
+  if (kind(node) == "private-attrib" || kind(node) == "private-func") {
+    return false;
+  }
+  // We do not generate documents for types in the C++ `std::` namespace:
+  auto const id = std::string_view{node.attribute("id").as_string()};
+  for (std::string_view prefix : {"namespacestd", "classstd", "structstd"}) {
+    if (id.substr(0, prefix.size()) == prefix) return false;
```

Comment:
why not `absl::StartsWith(id, prefix)` ?

Reply:
Because I am not using absl:: in this.  Dependencies are liabilities.